52/srisris/pomodoro/timer.py

```diff
@@ -0,0 +1,16 @@
+""" PCC52 - Create your own Pomodoro Timer 
+    Instructions + submit work: https://codechalleng.es/challenges/52"""
+
+from datetime import datetime, timedelta
+
+now = datetime.now()
+interval = timedelta(minutes=1)
+end_time = now + interval
+print(now)
+print(end_time)
+
+stopFlag = False
+while stopFlag == False:  
```

Comment:
while will tests much much faster then what you need for a timer, so it would be a good idea to use some `time.sleep` or equal methods to delay the code execution up to the next reasonable interval, like 1 second.